### ISSUE TITLE
Feature idea

### ISSUE THREAD

**Issue by JoeK1973** [starryalley/CSC_BLE_Bridge#19]
Hope this is the right place - I notice there isn't really an app that does ANT+ to BLE bridging for FE-C and power? Is that something that can implemented in future updates? Great work by the way, I've been thinking about how I would do somthing with the Nordic nRF52832 with Arduino or similar.

**Comment by starryalley** [comment]
Hi @JoeK1973 yes this is definitely possible. Just me or someone else has to implement it. I've seen there are ANT+ and BLE cycling power profiles so we can definitely add this in the future.

**Comment by mhanbury** [comment]
What a brilliant idea.  As above not sure if this is the right place but would it be possible to add support for other Ant+ profiles e.g. the running dynamics from the hrm-run or pod and temperature from the Tempe sensor?

**Comment by mhanbury** [comment]
I agree the priority should definitely be power. Temperature user case would be outdoor activities.

**Comment by bohtho** [comment]
I’ve also looked at all sort of ant+ BLE bridges (barring purchasing a Garmin device and diving into that limited rabbit hole), and also into doing it with arduino or rp0w, but seems it would be most elegant with an ant+ enabled android phone where you also can see maps on a very decent screen.

I’m specifically looking to get the ant+ LEV profile data that is emitted from lots of electric (mountain)bikes since 2020 bridged into BLE for use in (custom) apps. Part of this is of course normal power and cadence, but also battery remaining and assist mode (etc.).

I currently have no android phone but would quickly get one of the listed ones (have a
Nokia 7 plus on hand) to test if someone was able to chalk something up for this. I think it would be popular versus having to get a Garmin device, or use proprietary apps and expensive stupid displays.

**Comment by leaskovski** [comment]
So my old Nokia 7 power socket died, and I now have a new phone that doesn't have Ant+ anymore, and I don't want to worry about having a dongle hanging from it in my jersey pocket, as this is likely to bugger the new phones power socket.  The power meter on my road bike is Ant+ only, so am dead keen on getting Power implemented.

Whats the best way for me to get started on trying to get power profiles implemented in this?

**Comment by AlEEEX777** [comment]
Hello! Thank you for your application, you are just a genius!
My idea is to be able to organize a BLE-BLE bridge, so that a heart sensor such as Polar h9 with one BLE channel, connected to a smartphone without ANT+, could transmit to the GT Runner or other watch (without ANT+) via BLE, and at the same time Polar Beat application on the smartphone also continued to work with the sensor.

**Comment by diessence** [comment]
Hi guys, just seeing if anyone actually started the implementation of cycling power data. If not I'm keen to try and get this added so I'm willing to have a look at the source and see how hard it would be to add. Cheers, diessence

**Comment by starryalley** [comment]
Hi, I haven't worked on this project for a while. Please feel free to work on it :) Thanks!
